src/utils/swapData.ts

```diff
@@ -159,6 +171,20 @@ export function swapDataFrom0xQuote(
   }
 }
 
+function swapDataFromBalancer(
+  order: ZeroExApiSwapResponseOrderBalancer
+): SwapData | null {
+  const fillData = order.fillData
+  if (!fillData) return null
+  return {
+    exchange: Exchange.BalancerV2,
+    path: fillData.assets,
+    fees: [],
+    // FIXME: check
+    pool: fillData.vault,
```

Comment:
Should be checked again if this is the correct address you need.

Also, if `fees` should be populated. @ElmoArmy